atlassian_jwt_auth/verifier.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, public_key_retriever, **kwargs):
         self._seen_jti = OrderedDict()
         self._subject_should_match_issuer = kwargs.get(
             'subject_should_match_issuer', True)
+        self._check_jti_uniqueness = kwargs.get(
```

Comment:
I think if we continue going down the `kwargs` route, this begins to get hairy from a consumer point of view because you actually have to dive in here and read the code to discover how to properly call the function. I'd much rather eliminate `**kwargs` here and see something like:

```
def __init__(self, retriever, subject_should_match_issuer=True, check_jti_uniqueness=True):
    ...
```

Reply:
That's true but I think if we document the keyword arguments it can take it might be okay.